composeApp/src/commonMain/kotlin/zed/rainxch/githubstore/core/data/repository/StarredRepositoryImpl.kt

```
@@ -0,0 +1,196 @@
+            val relevantAssets = stableRelease.assets.filter { asset ->
+                val name = asset.name.lowercase()
+                when (platform.type) {
+                    PlatformType.ANDROID -> name.endsWith(".apk")
+                    PlatformType.WINDOWS -> name.endsWith(".msi") || name.endsWith(".exe") || name.contains(".exe")
+                    PlatformType.MACOS -> name.endsWith(".dmg") || name.endsWith(".pkg")
+                    PlatformType.LINUX -> name.endsWith(".appimage") || name.endsWith(".deb") || name.endsWith(".rpm")
+                }
+            }
+
+            relevantAssets.isNotEmpty()
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)